subsys/bluetooth/common/Kconfig

```diff
@@ -118,7 +118,39 @@ config BT_DEBUG_MONITOR
 	  UART_CONSOLE needs to be disabled (in which case printk/printf
 	  will get encoded into the monitor protocol).
 
-endchoice
+config BT_DEBUG_MONITOR_RTT
+	bool "Monitor protocol over RTT"
+	select BT_DEBUG
```

Comment:
Is the RTT functions always available? Otherwise you would need some depends on here, maybe USE_SEGGER_RTT?